src/allotropy/parsers/moldev_softmax_pro/softmax_pro_structure.py

```diff
@@ -390,7 +390,7 @@ def create(
 
         # convert rows to df
         data = assert_not_none(
-            reader.lines_as_df(lines=lines, sep="\t"),
+            reader.lines_as_df(lines=lines, sep="\t", header=None),
```

Comment:
Why are all these `header=None` necessary now? that seems like an undesirable change.

Reply:
The header=None is necessary if the kwargs approach is implemented. 
However I think that is a good thing that the missing header is explicitly declared.